### PR TITLE
chore: packs page DOC-807

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR carries over the work @Sheldonwl started in #1389. @Sheldonwl the original PR was created under the previous versions of docs. To avoid merge conflicts and issues, I ported the content into this branch and touched up the language. 

CleanShot 2023-11-08 at 15 40 25
 

## Review Changes

💻 [Preview URL](https://deploy-preview-1773--docs-spectrocloud.netlify.app/integrations/trident)

🎫 [DOC-807](https://spectrocloud.atlassian.net/browse/DOC-807)

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
Closes #1389

**Review by lennessyy — APPROVED** [review]
This looks good - but don't we need a Terraform section?

File: docs/docs-content/integrations/trident.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    requests:
+      storage: 1Gi
+  storageClassName: trident-csi
+```
+
+<br />
+
+#### Deploy a Pod and Mount the Volume.  
+
+The following code snippet is an example of a pod deployment. The Pod contains an Nginx container, with a claim to the previously created PVC. Once deployed, the PVC with its bound Persistent Volume (PV), will mount to the Pod and provide persistent storage to this application.
```
Comment (lennessyy) [inline docs/docs-content/integrations/trident.md:337]:
```suggestion
The following code snippet is an example of a pod deployment. The pod contains an Nginx container, with a claim to the previously created PVC. Once deployed, the PVC with its bound Persistent Volume (PV), will mount to the Pod and provide persistent storage to this application.    
```

File: docs/docs-content/integrations/trident.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
```
Comment (lennessyy) [inline docs/docs-content/integrations/trident.md:167]:
```suggestion
The following code snippet is an example of a pod deployment. The pod contains an Nginx container, with a claim to the previously created PVC. Once deployed, the PVC with its bound Persistent Volume (PV), will mount to the pod and provide persistent storage to this application.    
```

File: docs/docs-content/integrations/trident.md [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+  svm: [SVM]
+  credentials: 
+    name: backend-ontap-nas-secret
+```
+
+
+
+#### Create a Storage Class 
+
+Kubernetes supports the ability to bind statically or dynamically provisioned volumes to Pods. Statically provisioned volumes are manually created by a user and then referenced in a deployment. Astra Trident allows you to leverage your NetApp storage. You must create a storage class before you can request dynamically provisioned volumes.
```
Comment (lennessyy) [inline docs/docs-content/integrations/trident.md:126]:
```suggestion
Kubernetes supports the ability to bind statically or dynamically provisioned volumes to pods. Statically provisioned volumes are manually created by a user and then referenced in a deployment. Astra Trident allows you to leverage your NetApp storage. You must create a storage class before you can request dynamically provisioned volumes. 
```

**Pull request merged**